.github/workflows/ci-pipeline.yml

```
@@ -1,12 +1,7 @@
-name: Unitify CI/CD Pipeline
+name: Unitify CI Pipeline
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)